modules/media/test/TestPlan.md

```diff
@@ -82,13 +82,13 @@ is selected, the file name should should start with [PSCID]\_[Visit Label]\_[Ins
     - _Note: only a person with direct database access could revert the hide file action._
 
 **Test filters** [Automation Testing]
-  1. Under **Browse** tab, a selection filter should be present on top of the page containing the following fields: File Name, Language, Project, File Visibility, PSCID, Instrument, Uploaded By, Visit Label, Site and File type.
+  1. Under **Browse** tab, a selection filter should be present on top of the page containing the following fields: File Name, Language, Project, File Visibility, PSCID, Instrument, Uploaded By, Visit Label, and File type.
```

Comment:
Hi @haoweiqiu, I think for filters `Site` exists.
![media_table](https://user-images.githubusercontent.com/16293415/93641733-c8745a80-f9ca-11ea-9056-aa2e391b486f.jpg)